cmd/tegola/cmd/cache.go

```diff
@@ -248,7 +248,7 @@ func SeedWorker(ctx context.Context, mt MapTile) error {
 	//	https://github.com/golang/go/issues/14045 - should be addressed in Go 1.11
 	runtime.GC()
 
-	log.Infof("seeding map (%v) tile (%v/%v/%v) took: %v", mt.MapName, z, x, y, time.Now().Sub(t))
+	log.Infof("seeding map (%v) tile (%v/%v/%v) took: %vms", mt.MapName, z, x, y, time.Now().Sub(t).Nanoseconds()/1000)
```

Comment:
That's microseconds. Not sure if this precision in necessary :-)
Either Nanoseconds()/1000000 or Seconds()*1000 and %d as format.

Reply:
@olt great catch! 🙏